Assets/Mirror/Components/NetworkPingDisplay.cs

```diff
@@ -14,6 +14,8 @@ public class NetworkPingDisplay : MonoBehaviour
         [SerializeField] Vector2 position = new Vector2(200, 0);
         [SerializeField] int fontSize = 24;
         [SerializeField] Color textColor = new Color32(255, 255, 255, 80);
+        [SerializeField] string textPrefix;
```

Comment:
please make them public (so we don't need the extra update methods) and name them 'prefix' and 'suffix' for simplicity :)